native/packages/react-native-bpk-component-banner-alert/src/AnimateAndFade.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Animated,
+  ViewPropTypes,
+} from 'react-native';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import BpkAnimateHeight from 'react-native-bpk-component-animate-height';
+import { animationDurationSm } from 'bpk-tokens/tokens/base.react.native';
+
+const fadeDuration = parseInt(animationDurationSm, 10);
```

Comment:
`bpk-tokens` transforms duration tokens into types digestible from RN now, so you don't need to do `parseInt` anymore. See [here](https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/tokens/ios/base.react.native.es6.js#L21).